src/sctools/metrics/gatherer.py

```diff
@@ -158,6 +207,60 @@ def extract_metrics(self, mode: str = "rb") -> None:
                 )
                 cell_metrics_output.write(cell_tag, vars(metric_aggregator))
 
+class GatherCellMetricsFast(MetricGatherer):
```

Comment:
Was there a non-fast version of this method that can be replaced by this?

Reply:
removed this misleading naming to reflect that it parses from tsv file.

Reply:
this also appears unchanged. Also do we use this code or is the C++ code being invoked directly?